Gemfile.lock

```diff
@@ -116,7 +116,8 @@ GEM
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (1.0.0)
-    mimemagic (0.3.5)
+    mimemagic (0.3.8)
```

Comment:
Is this change related?

Reply:
this was needed for CI to work (and to run test locally 😓 )